### ISSUE TITLE
Latest Windows Builds

### ISSUE THREAD

**Issue by DropdeadFredd** [iliaal/php_excel#157]
Hi

Do you have a link to the latest windows builds?

The readme links to builds dated: 2014-06-06 but the Change log has a 1.0.1 version from 2015-02-26.

Thanks, Fred

**Comment by johmue** [comment]
Have a look here https://www.apachelounge.com/viewtopic.php?t=6617 - plenty of other pre-compiled extensions for php ...

**Comment by DropdeadFredd** [comment]
Cool thanks.

I also found this -> https://github.com/johmue/win-php-sdk-builder

Which seem really useful, unfortunately my windows environment is messed up and I couldn't get it to compile and don't have the time to sort it out at the moment.  Maybe others will have better luck.

**Comment by johmue** [comment]
@DropdeadFredd would be cool if you could tell me what did not work since I'm the maintainer of this project... :)

**Comment by DropdeadFredd** [comment]
No problem, but I don't think the issue was with your project, rather my windows environment that has had multiple versions of Visual Studio and other software installed and un-installed. And we all know how good Windows is at cleaning up after itself.

I had a couple of issues:

Initially the build batch file couldn't find the wget or 7za executables and failed to download them. I tried downloading them and adding them to my path but it still didn't work. Eventually I put them both in my Windows folder and then build ran.

When running the build wget crashed downloading the PHP Dependancy file: deps-5.5-vc11-x86.7z but the file appeared to have downloaded OK and the build continued. (I've since downloaded the file manually and re-run the build to ensure that there wasn't a problem with the archive).

When I run the generated compile batch file I get the following:

> Microsoft (R) Program Maintenance Utility Version 11.00.50727.1
> Copyright (C) Microsoft Corporation.  All rights reserved.
> 
> NMAKE : fatal error U1073: don't know how to make 'clean'
> Stop.
> Input Error: There is no script engine for file extension ".js".
> Now run 'configure --help'
> Input Error: There is no script engine for file extension ".js".
> 
> Microsoft (R) Program Maintenance Utility Version 11.00.50727.1
> Copyright (C) Microsoft Corporation.  All rights reserved.
> 
> NMAKE : fatal error U1073: don't know how to make 'snap'
> Stop.
> Press any key to continue . . .

For the first "Input Error: There is no script engine for the file extension .js" I added " /e:jscript" to 

> cscript /nologo win32\build\buildconf.js %*

in buildconf.bat as per the comment here -> http://php.net/manual/en/install.windows.building.php.

> cscript /nologo  /e:jscript win32\build\buildconf.js %*

Which appeared to fix that one error.

I couldn't find a solution to the clean error.

The script engine error appears to be an issue with my .js file association. I've tried to reset it but it still doesn't work.

I presume the NMAKE snap error is because of the previous script engine error.

If you want to pursue this and need any more info just let me know.
